src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -137,29 +151,25 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
         reserveMutation.data.url,
         'cancel',
         `${APP_SCHEME}://purchase-callback`,
-        navigateToActiveTicketsScreen,
+        onPaymentCompleted,
       );
     }
   }, [
     reserveMutation.isSuccess,
     reserveMutation.data?.url,
     paymentMethod?.paymentType,
-    navigateToActiveTicketsScreen,
+    onPaymentCompleted,
   ]);
 
   // When deep link {APP_SCHEME}://purchase-callback is called, save payment
   // method and navigate to active tickets.
-  usePurchaseCallbackListener(
-    navigateToActiveTicketsScreen,
-    paymentMethod?.paymentType,
-    reserveMutation.data?.recurring_payment_id,
-  );
+  usePurchaseCallbackListener(onPaymentCompleted);
```

Comment:
Do we really need this listener at all anymore? 🤔

Reply:
Yes, Vipps payment still uses it.